src/main/riddl/Venues/Reservations/reservationMessages.riddl

```diff
@@ -0,0 +1,45 @@
+command EstablishReservation is {
+  reservationId: ReservationId briefly "A unique identifier for this Reservation. This must be provided by the calling client.",
+  info: ReservationContext.Info briefly "Details about a Reservation required for it to exist. See ReservationContext.Info for details.",
+  establishingMember: MemberId briefly "The Id of the Member who is issuing the create command.",
+  owner: MemberId? briefly "Used in the case that establishingMember is not the owner"
+} briefly "Creates a Reservation"
+
+event ReservationEstablished is {
+  reservationId: ReservationId,
+  info: ReservationContext.Info,
+  meta: ReservationContext.MetaInfo
+}
+
+command EditReservationInfo is {
+  venueId: VenueId briefly "The unique Id of the reservation to create",
+  newInfo: ReservationContext.UpdateInfo briefly "The information about the reservation",
+  editingMember: MemberId briefly "Id of the member who is creating the reservation"
+} briefly "A command to edit the values of an Reservation's Info "
+event ReservationInfoUpdated is {
+  reservationId: ReservationId,
+  info: ReservationContext.Info,
+  meta: ReservationContext.MetaInfo
+}
+
+command UpdateReservationStatus is {
+  reservationId: ReservationId,
+  status: ReservationContext.Status,
+  updatingMember: MemberId
+}
+event ReservationStatusUpdated is {
+  reservationId: ReservationId,
+  newStatus: ReservationContext.Status
+}
+
+command AddReservationAttendees is {
+  reservationId: ReservationId,
+  attendees: MemberId+
+}
+event ReservationAttendeesAdded is {
+  venueId: VenueId,
+  allAttendees: MemberId+
+}
+
+command DeleteReservation is {reservationId: ReservationId}
```

Comment:
CancelReservation/ReservationCancelled?

Reply:
Outdated - not needed because we can just use UpdateStatus